### DISCUSSION TITLE
Make a cluster scatter plot from distance matrix

### DISCUSSION THREAD

**Discussion by JudKil** [rdkit/rdkit#5380] [General]
Hello everyone,

My name is Judith and for my PhD studies,  I would like to use rdkit and pandas to manipulate ligand poses.

I get a distance matrix by rmsd between each pose but I don't see how to pass it to a scatter plot of 2 clusters, I tried with pandas but I'm really blocked, I can't select the lines and the columns to generate the scatter plot

Best Regards,
Judith

**Comment by greglandrum** [comment]
@JudKil What do you mean by "scatter plot of 2 clusters"? What would be on the X and Y axes of that scatter plot? Can you provide an example (i.e link or image) of the kind of plot you are trying to make?
